scripts/quests/windurst/Carbuncle_Debacle.lua

```diff
@@ -0,0 +1,186 @@
+}
+
+quest.sections =
+{
+    --Section: Quest Available
+
+    {
+
+        check = function(player, status, vars)
+            return status == QUEST_AVAILABLE and
+                player:getMainLvl() >= xi.settings.main.AF3_QUEST_LEVEL and
+                player:getMainJob() == xi.job.SMN and
+                not quest:getMustZone(player)
+        end,
+
+        [xi.zone.WINDURST_WALLS] =
+        {
+            ['_6n2'] = quest:progressEvent(415),
+
+                onEventFinish =
+                {
+                    [415] = function(player, csid, option, npc)
+                            quest:begin(player)
+                            quest:setVar(player, 'Prog', 1)
+                    end,
+                },
+        },
+    },
+
+    {
+        check = function(player, status, vars)
+            return status == QUEST_ACCEPTED
+        end,
+
+        --Section: Quest Accepted
+        [xi.zone.WINDURST_WALLS] =
+        {
+            ['Koru-Moru'] =
+            {
+                onTrigger = function(player, npc)
+                    local questProgress = quest:getVar(player, 'Prog')
+
+                    if questProgress == 1 then
+                        return quest:progressEvent(416)
+                    elseif questProgress == 4 then
+                        return quest:progressEvent(417)
+                    elseif questProgress == 5 then
+                        return quest:event(418)
+                    elseif questProgress == 7 then
+                        return quest:progressEvent(419)
+                    elseif player:getQuestStatus(xi.quest.log_id.WINDURST, xi.quest.id.windurst.CARBUNCLE_DEBACLE) == QUEST_COMPLETED then
+                        return quest:event(420)
+                    end
+                end,
+            },
+                onEventFinish =
+                {
+                    [416] = function(player, csid, option, npc)
+                        quest:setVar(player, 'Prog', 2)
+                    end,
+
+                    [417] = function(player, csid, option, npc)
+                        quest:setVar(player, 'Prog', 5)
+                        npcUtil.giveKeyItem(player, xi.ki.DAZE_BREAKER_CHARM)
+                    end,
+
+                    [419] = function(player, csid, option, npc)
+                        quest:complete(player)
+                        player:addTitle(xi.title.PARAGON_OF_SUMMONER_EXCELLENCE)
+                    end
+                }
+            },
+
+        [xi.zone.MHAURA] =
+        {
+
+            ['Ripapa'] =
+            {
+                onTrigger = function(player, npc)
+                    local questProgress = quest:getVar(player, 'Prog')
+
+                    if questProgress == 2 then
+                        return quest:progressEvent(10022)
+                    elseif questProgress == 3 and not player:hasItem(xi.items.LIGHTNING_PENDULUM) then
+                        return quest:progressEvent(10023)
```

Comment:
`elseif 
    questProgress == 3 and
   not player:hastItem(xi.items.LIGHTNING_PENDULUM)
then
    return quest:progressEvent(10023)`